The test `test_vlob_group_updated_event` is inconsistent

Seen in this [appveyor build](https://ci.appveyor.com/project/touilleMan/parsec-cloud/builds/22719006):
```python-traceback
________________________ test_vlob_group_updated_event ________________________
[gw0] win32 -- Python 3.6.8 c:\python36-x64\python.exe
value = <trio._core._run.Nursery object at 0x00000054D197FB70>
    async def yield_(value=None):
>       return await _yield_(value)
c:\python36-x64\lib\site-packages\async_generator\_impl.py:106: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
c:\python36-x64\lib\site-packages\async_generator\_impl.py:99: in _yield_
    return (yield _wrap(value))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
backend = <parsec.backend.app.BackendApp object at 0x00000054D195DCC0>
alice_backend_sock = <tests.common.FreezeTestOnTransportError object at 0x00000054D3095160>
alice = LocalDevice(alice@dev1), alice2 = LocalDevice(alice@dev2)
    @pytest.mark.trio
    async def test_vlob_group_updated_event(backend, alice_backend_sock, alice, alice2):
        # Not listened events
    
        await backend.vlob.create(alice.organization_id, alice.device_id, GROUP_ID, VLOB_ID, b"v1")
    
        # Start listening events
    
        await events_subscribe(alice_backend_sock, vlob_group_updated=[GROUP_ID, OTHER_GROUP_ID])
    
        # Good events
    
        await backend.vlob.create(
            alice.organization_id, alice2.device_id, OTHER_GROUP_ID, OTHER_VLOB_ID, b"v1"
        )
        await backend.vlob.update(alice.organization_id, alice2.device_id, VLOB_ID, 2, b"v2")
        await backend.vlob.update(alice.organization_id, alice2.device_id, VLOB_ID, 3, b"v3")
    
        with trio.fail_after(1):
            # No guarantees those events occur before the commands' return
            await backend.event_bus.spy.wait_multiple(
                ["vlob_group.updated", "vlob_group.updated", "vlob_group.updated"]
            )
    
        reps = [
            await events_listen_nowait(alice_backend_sock),
            await events_listen_nowait(alice_backend_sock),
            await events_listen_nowait(alice_backend_sock),
            await events_listen_nowait(alice_backend_sock),
        ]
>       assert reps == [
            {
                "status": "ok",
                "event": "vlob_group.updated",
                "id": OTHER_GROUP_ID,
                "checkpoint": 1,
                "src_id": OTHER_VLOB_ID,
                "src_version": 1,
            },
            {
                "status": "ok",
                "event": "vlob_group.updated",
                "id": GROUP_ID,
                "checkpoint": 2,
                "src_id": VLOB_ID,
                "src_version": 2,
            },
            {
                "status": "ok",
                "event": "vlob_group.updated",
                "id": GROUP_ID,
                "checkpoint": 3,
                "src_id": VLOB_ID,
                "src_version": 3,
            },
            {"status": "no_events"},
        ]
E       AssertionError: assert [{'checkpoint... 'no_events'}] == [{'checkpoint'... 'no_events'}]
E         At index 2 diff: {'status': 'no_events'} != {'status': 'ok', 'event': 'vlob_group.updated', 'id': UUID('00000000-0000-0000-0000-00000000000a'), 'checkpoint': 3, 'src_id': UUID('00000000-0000-0000-0000-000000000001'), 'src_version': 3}
E         Full diff:
E         [{'checkpoint': 1,
E         'event': 'vlob_group.updated',
E         'id': UUID('00000000-0000-0000-0000-00000000000b'),
E         'src_id': UUID('00000000-0000-0000-0000-000000000002'),
E         'src_version': 1,
E         'status': 'ok'},
E         {'checkpoint': 2,
E         'event': 'vlob_group.updated',
E         'id': UUID('00000000-0000-0000-0000-00000000000a'),
E         'src_id': UUID('00000000-0000-0000-0000-000000000001'),
E         'src_version': 2,
E         'status': 'ok'},
E         -  {'status': 'no_events'},
E         +  {'checkpoint': 3,
E         +   'event': 'vlob_group.updated',
E         +   'id': UUID('00000000-0000-0000-0000-00000000000a'),
E         +   'src_id': UUID('00000000-0000-0000-0000-000000000001'),
E         +   'src_version': 3,
E         +   'status': 'ok'},
E         {'status': 'no_events'}]
tests\backend\test_vlob_group.py:267: AssertionError
---------------------------- Captured stdout call -----------------------------
2019-02-28 13:40.04 notif sent                     kwargs={'organization_id': <OrganizationID 'CoolOrg'>, 'author': <DeviceID 'alice@dev1'>, 'id': UUID('3289831b-7ad1-4563-8e72-0f181fc4784f'), 'checkpoint': 1, 'src_id': UUID('3289831b-7ad1-4563-8e72-0f181fc4784f'), 'src_version': 1} signal=vlob_group.updated
2019-02-28 13:40.04 notif received                 channel=app_notification payload=iKZfX2lkX1/ZIGY2YjQxYTNkODJhZjQ1N2ZhNjdmMzAzYzQyZjIxZWE5ql9fc2lnbmFsX1+ydmxvYl9ncm91cC51cGRhdGVkr29yZ2FuaXphdGlvbl9pZKdDb29sT3JnpmF1dGhvcqphbGljZUBkZXYxomlk2AIyiYMbetFFY45yDxgfxHhPqmNoZWNrcG9pbnQBpnNyY19pZNgCMomDG3rRRWOOcg8YH8R4T6tzcmNfdmVyc2lvbgE= pid=3620
2019-02-28 13:40.04 notif sent                     kwargs={'organization_id': <OrganizationID 'OtherOrg'>, 'author': <DeviceID 'alice@dev1'>, 'id': UUID('c6a18cab-406b-46c3-a05a-bcb8489861ef'), 'checkpoint': 1, 'src_id': UUID('c6a18cab-406b-46c3-a05a-bcb8489861ef'), 'src_version': 1} signal=vlob_group.updated
2019-02-28 13:40.04 notif received                 channel=app_notification payload=iKZfX2lkX1/ZIGVmYzg2ZTVlMDRhZjRlNTA5NjUwMzhjYjkwNTIyZWFjql9fc2lnbmFsX1+ydmxvYl9ncm91cC51cGRhdGVkr29yZ2FuaXphdGlvbl9pZKhPdGhlck9yZ6ZhdXRob3KqYWxpY2VAZGV2MaJpZNgCxqGMq0BrRsOgWry4SJhh76pjaGVja3BvaW50AaZzcmNfaWTYAsahjKtAa0bDoFq8uEiYYe+rc3JjX3ZlcnNpb24B pid=3620
2019-02-28 13:40.05 notif sent                     kwargs={'organization_id': <OrganizationID 'CoolOrg'>, 'device_id': <DeviceID 'alice@dev2'>, 'encrypted_answer': b''} signal=device.created
2019-02-28 13:40.05 notif received                 channel=app_notification payload=haZfX2lkX1/ZIDdmM2MxZTdhNzkzNTQyMDNhNjY4ZjIxODkwYjRjMTlkql9fc2lnbmFsX1+uZGV2aWNlLmNyZWF0ZWSvb3JnYW5pemF0aW9uX2lkp0Nvb2xPcmepZGV2aWNlX2lkqmFsaWNlQGRldjKwZW5jcnlwdGVkX2Fuc3dlcsQA pid=3620
2019-02-28 13:40.05 notif sent                     kwargs={'organization_id': <OrganizationID 'CoolOrg'>, 'user_id': <UserID 'bob'>} signal=user.created
2019-02-28 13:40.05 notif sent                     kwargs={'organization_id': <OrganizationID 'CoolOrg'>, 'author': <DeviceID 'bob@dev1'>, 'id': UUID('e5b63292-b460-43fe-93b5-14b1bedca171'), 'checkpoint': 1, 'src_id': UUID('e5b63292-b460-43fe-93b5-14b1bedca171'), 'src_version': 1} signal=vlob_group.updated
2019-02-28 13:40.05 notif received                 channel=app_notification payload=hKZfX2lkX1/ZIGE1NzI3MzljYTU1ODRkM2ZhYWI5MzMwZGQyZmViYzVlql9fc2lnbmFsX1+sdXNlci5jcmVhdGVkr29yZ2FuaXphdGlvbl9pZKdDb29sT3Jnp3VzZXJfaWSjYm9i pid=3620
2019-02-28 13:40.05 Accepting WebSocket upgrade    conn_id=8785708ed447454ba298c5ad775acb70
2019-02-28 13:40.05 Client joined                  conn_id=8785708ed447454ba298c5ad775acb70
2019-02-28 13:40.05 start handshake                conn_id=8785708ed447454ba298c5ad775acb70
2019-02-28 13:40.05 WebSocket negotiation complete conn_id=fcaa0b6ab9cb4abcafd187356b48a82c ws_event=<ConnectionEstablished subprotocol=None extensions=[]>
2019-02-28 13:40.05 notif received                 channel=app_notification payload=iKZfX2lkX1/ZIGJlMWQyOGZhYjk3ZTRiZTNhYzczNjg4ZGQyYmVkOTBjql9fc2lnbmFsX1+ydmxvYl9ncm91cC51cGRhdGVkr29yZ2FuaXphdGlvbl9pZKdDb29sT3JnpmF1dGhvcqhib2JAZGV2MaJpZNgC5bYykrRgQ/6TtRSxvtyhcapjaGVja3BvaW50AaZzcmNfaWTYAuW2MpK0YEP+k7UUsb7coXGrc3JjX3ZlcnNpb24B pid=3620
2019-02-28 13:40.05 handshake done                 client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg
2019-02-28 13:40.05 Client handshake done          client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg
2019-02-28 13:40.05 notif sent                     kwargs={'organization_id': <OrganizationID 'CoolOrg'>, 'author': <DeviceID 'alice@dev1'>, 'id': UUID('00000000-0000-0000-0000-00000000000a'), 'checkpoint': 1, 'src_id': UUID('00000000-0000-0000-0000-000000000001'), 'src_version': 1} signal=vlob_group.updated
2019-02-28 13:40.05 req                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg req={'cmd': 'events_subscribe', 'vlob_group_updated': [UUID('00000000-0000-0000-0000-00000000000a'), UUID('00000000-0000-0000-0000-00000000000b')]}
2019-02-28 13:40.05 Request                        client_id=alice@dev1 cmd=events_subscribe conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg
2019-02-28 13:40.05 rep                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg rep={'status': 'ok'}
2019-02-28 13:40.05 notif received                 channel=app_notification payload=iKZfX2lkX1/ZIGIyZmI1MGFkMThjYTRlNDFhMTgzNjJiODcwNGRlMTM2ql9fc2lnbmFsX1+ydmxvYl9ncm91cC51cGRhdGVkr29yZ2FuaXphdGlvbl9pZKdDb29sT3JnpmF1dGhvcqphbGljZUBkZXYxomlk2AIAAAAAAAAAAAAAAAAAAAAKqmNoZWNrcG9pbnQBpnNyY19pZNgCAAAAAAAAAAAAAAAAAAAAAatzcmNfdmVyc2lvbgE= pid=3620
2019-02-28 13:40.05 notif sent                     kwargs={'organization_id': <OrganizationID 'CoolOrg'>, 'author': <DeviceID 'alice@dev2'>, 'id': UUID('00000000-0000-0000-0000-00000000000b'), 'checkpoint': 1, 'src_id': UUID('00000000-0000-0000-0000-000000000002'), 'src_version': 1} signal=vlob_group.updated
2019-02-28 13:40.05 notif received                 channel=app_notification payload=iKZfX2lkX1/ZIDAyNmMyMGRmMjRhMDRlN2M5ZjFiMjk3YWYzYzM3Mzdiql9fc2lnbmFsX1+ydmxvYl9ncm91cC51cGRhdGVkr29yZ2FuaXphdGlvbl9pZKdDb29sT3JnpmF1dGhvcqphbGljZUBkZXYyomlk2AIAAAAAAAAAAAAAAAAAAAALqmNoZWNrcG9pbnQBpnNyY19pZNgCAAAAAAAAAAAAAAAAAAAAAqtzcmNfdmVyc2lvbgE= pid=3620
2019-02-28 13:40.05 notif sent                     kwargs={'organization_id': <OrganizationID 'CoolOrg'>, 'author': <DeviceID 'alice@dev2'>, 'id': UUID('00000000-0000-0000-0000-00000000000a'), 'checkpoint': 2, 'src_id': UUID('00000000-0000-0000-0000-000000000001'), 'src_version': 2} signal=vlob_group.updated
2019-02-28 13:40.05 notif received                 channel=app_notification payload=iKZfX2lkX1/ZIDQxOTlmYWQ5M2E5MzRhNzg5NjViZWYyNGNjMDMyMGU3ql9fc2lnbmFsX1+ydmxvYl9ncm91cC51cGRhdGVkr29yZ2FuaXphdGlvbl9pZKdDb29sT3JnpmF1dGhvcqphbGljZUBkZXYyomlk2AIAAAAAAAAAAAAAAAAAAAAKqmNoZWNrcG9pbnQCpnNyY19pZNgCAAAAAAAAAAAAAAAAAAAAAatzcmNfdmVyc2lvbgI= pid=3620
2019-02-28 13:40.05 notif sent                     kwargs={'organization_id': <OrganizationID 'CoolOrg'>, 'author': <DeviceID 'alice@dev2'>, 'id': UUID('00000000-0000-0000-0000-00000000000a'), 'checkpoint': 3, 'src_id': UUID('00000000-0000-0000-0000-000000000001'), 'src_version': 3} signal=vlob_group.updated
2019-02-28 13:40.05 req                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg req={'cmd': 'events_listen', 'wait': False}
2019-02-28 13:40.05 Request                        client_id=alice@dev1 cmd=events_listen conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg
2019-02-28 13:40.05 rep                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg rep={'checkpoint': 1, 'event': 'vlob_group.updated', 'id': UUID('00000000-0000-0000-0000-00000000000b'), 'src_id': UUID('00000000-0000-0000-0000-000000000002'), 'src_version': 1, 'status': 'ok'}
2019-02-28 13:40.05 req                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg req={'cmd': 'events_listen', 'wait': False}
2019-02-28 13:40.05 Request                        client_id=alice@dev1 cmd=events_listen conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg
2019-02-28 13:40.05 rep                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg rep={'checkpoint': 2, 'event': 'vlob_group.updated', 'id': UUID('00000000-0000-0000-0000-00000000000a'), 'src_id': UUID('00000000-0000-0000-0000-000000000001'), 'src_version': 2, 'status': 'ok'}
2019-02-28 13:40.05 req                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg req={'cmd': 'events_listen', 'wait': False}
2019-02-28 13:40.05 Request                        client_id=alice@dev1 cmd=events_listen conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg
2019-02-28 13:40.05 rep                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg rep={'status': 'no_events'}
2019-02-28 13:40.05 req                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg req={'cmd': 'events_listen', 'wait': False}
2019-02-28 13:40.05 Request                        client_id=alice@dev1 cmd=events_listen conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg
2019-02-28 13:40.05 rep                            client_id=alice@dev1 conn_id=8785708ed447454ba298c5ad775acb70 organization_id=CoolOrg rep={'status': 'no_events'}
2019-02-28 13:40.05 notif received                 channel=app_notification payload=iKZfX2lkX1/ZIGYzYzI0YTNhNTA1MDQ2YmY5ODdiNjc0N2QxOTY1NDA3ql9fc2lnbmFsX1+ydmxvYl9ncm91cC51cGRhdGVkr29yZ2FuaXphdGlvbl9pZKdDb29sT3JnpmF1dGhvcqphbGljZUBkZXYyomlk2AIAAAAAAAAAAAAAAAAAAAAKqmNoZWNrcG9pbnQDpnNyY19pZNgCAAAAAAAAAAAAAAAAAAAAAatzcmNfdmVyc2lvbgM= pid=3620
```

--- a/tests/backend/test_vlob_group.py
+++ b/tests/backend/test_vlob_group.py
@@ -236,27 +236,26 @@ async def test_vlob_group_handle_rights(backend, alice, bob, alice_backend_sock,
 
 @pytest.mark.trio
 async def test_vlob_group_updated_event(backend, alice_backend_sock, alice, alice2):
-    # Not listened events
-
+    # Not listened event
     await backend.vlob.create(alice.organization_id, alice.device_id, GROUP_ID, VLOB_ID, b"v1")
 
     # Start listening events
-
     await events_subscribe(alice_backend_sock, vlob_group_updated=[GROUP_ID, OTHER_GROUP_ID])
 
     # Good events
+    with backend.event_bus.listen() as spy:
 
-    await backend.vlob.create(
-        alice.organization_id, alice2.device_id, OTHER_GROUP_ID, OTHER_VLOB_ID, b"v1"
-    )
-    await backend.vlob.update(alice.organization_id, alice2.device_id, VLOB_ID, 2, b"v2")
-    await backend.vlob.update(alice.organization_id, alice2.device_id, VLOB_ID, 3, b"v3")
-
-    with trio.fail_after(1):
-        # No guarantees those events occur before the commands' return
-        await backend.event_bus.spy.wait_multiple(
-            ["vlob_group.updated", "vlob_group.updated", "vlob_group.updated"]
+        await backend.vlob.create(
+            alice.organization_id, alice2.device_id, OTHER_GROUP_ID, OTHER_VLOB_ID, b"v1"
         )
+        await backend.vlob.update(alice.organization_id, alice2.device_id, VLOB_ID, 2, b"v2")
+        await backend.vlob.update(alice.organization_id, alice2.device_id, VLOB_ID, 3, b"v3")
+
+        with trio.fail_after(1):
+            # No guarantees those events occur before the commands' return
+            await spy.wait_multiple(
+                ["vlob_group.updated", "vlob_group.updated", "vlob_group.updated"]
+            )
 
     reps = [
         await events_listen_nowait(alice_backend_sock),

